scripts/release/version.mjs

```diff
@@ -49,10 +58,22 @@ export async function version_handler(version) {
 		) {
 			continue;
 		}
-		const newManifest = {
-			...workspace.manifest,
-			version: nextVersion
-		};
+		let newManifest;
+
+		if (version === "snapshot") {
+			const nextName = getNextName(workspace.manifest.name);
+			newManifest = {
+				...workspace.manifest,
+				name: nextName,
+				version: nextVersion
+			};
+		} else {
+			newManifest = {
+				...workspace.manifest,
+				version: nextVersion
+			};
+		}
+		console.log(newManifest.name);
```

Comment:
should we delete this log

Reply:
I think it is okay to be preserved
<img width="400" alt="image" src="https://github.com/web-infra-dev/rspack/assets/79413249/5d1e6a31-9875-400e-a699-f8da399c3701">